js/storage.js

```diff
@@ -120,7 +120,7 @@ Storage.bg = {
 				break;
 			case 'psday':
 				hues = ["24.705882352941174,25.37313432835821%", "260.4651162790697,59.44700460829492%", "165.3191489361702,46.07843137254901%", "16.363636363636367,42.63565891472869%", "259.04761904761904,34.05405405405405%", "24.705882352941174,25.37313432835821%"];
-				attrib = '<a href="https://pokemonshowdown.com/news/216" target="_blank" class="subtle">Pokemon Showdown Day!</a>';
+				attrib = 'Pokemon Showdown Day Background!';
```

Comment:
Should probably credit the artist. I think it was LiD? We'll have to ask around.

Reply:
> dont use my full name ppl are still curious to where i am so like i dont want that. Just LifeisDANK nothing else needed I have no real online portfolio anymore.

Was what LiD responded with when you asked her about this, check #639

Reply:
Oh yeah I remember trying to figure out how to best implement that.

Reply:
```suggestion
				attrib = 'Pokémon Showdown Day background <small>by LifeisDANK</small>';
```

Reply:
`Pok&eacute;mon` with an escaped accent would be better, no?